### PR TITLE
refactor alerts to not use redux sauce & add pf4 auto hiding alert

### LINKED ISSUE #214: Success and failure toast notifications
According to https://github.com/fusor/mig-ui/pull/211#discussion_r289669972 we need to add success and failure [notifications](https://www.patternfly.org/pattern-library/communication/toast-notifications/) for such events as add/remove clusters, storage, plans, etc

### PR DESCRIPTION
**Pull request by ibolton336**
Closes #214 #62

### CODE REVIEW

**Comment by andyvesel** [comment]
@ibolton336 I think we also need to add success notifications here for remove cluster or storage, b/c for now, when delete a cluster/storage, nothing happens on the screen and it needs to manually refresh the page.

**Comment by ibolton336** [comment]
> @ibolton336 I think we also need to add success notifications here for remove cluster or storage, b/c for now, when delete a cluster/storage, nothing happens on the screen and it needs to manually refresh the page.

@andyvesel This PR will need to go in so that we can use the updated alert notification code on johns PR.

File: src/app/common/duck/actions.ts [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+export const ALERT_CLEAR = 'ALERT_CLEAR';
 
-const { Creators, Types } = createActions({
-  alertSuccess: ['success'],
-  alertError: ['alertMessage'],
-  alertClear: [],
-});
-
-export { Creators, Types };
+export function alertSuccess(text) {
```
Comment (eriknelson) [inline src/app/common/duck/actions.ts]: Can we enforce with TS that `text` here can only be a string? We've had multiple issues with `errors` getting shoved into these methods and it blowing up at runtime. It'd be great if we could lean on TS more to check this stuff when possible at compile time.

File: src/app/common/duck/operations.ts [hunk 2]
```diff
@@ -1,9 +1,31 @@
-import { Creators } from './actions';
-const alertSuccess = Creators.alertSuccess;
-const alertError = Creators.alertError;
-const alertClear = Creators.alertClear;
+import { alertSuccess, alertError, alertClear } from './actions';
+const alertErrorTimeout = message => {
```
Comment (eriknelson) [inline src/app/common/duck/operations.ts]: The arguments to these operations should also only accept TS strings, i.e. `message: String`.

**Pull request merged**